autoprotocol/harness.py

```diff
@@ -26,6 +26,23 @@
 }
 
 
+def get_protocol_preview(protocol, name):
```

Comment:
We haven't historically been good about documentation for things in harness, but would you mind adding some documentation here?

Some blurb about what the function does and an example would be awesome :)

Reply:
can do!